constants/geometries.js

```diff
@@ -158,7 +167,7 @@ export const geometries = [
     description:
       'The tick mark represents each data point as a short line. This is a useful mark for displaying the distribution of values in a field',
     defaultAesthetics: ['x', 'y'],
-    options: [],
+    options: [colorOption],
```

Comment:
I probably don't fully uinderstand what's going on here, but if defaultAesthetics doesn't have 'color' in does it need to be in options?

Reply:
Yeah its confusing, `defaultAesthetics` defines the aesthetics that appear in the list when the geometry type is selected. `options` is an additional set of options that are on the geometry and appear in the geometry list. `color` in the aesthetics list can be set to a variable to produce a separate mark on the geometry each with a different colour as per the variable but it can't be set to a static value. `color` in the options list can be set to a static value but can't be set to a variable, so they serve 2 different purposes.